packages/core/src/mesh/ModelMesh.ts

```diff
@@ -14,4 +65,685 @@ export class ModelMesh extends Mesh {
+   * @param weights - The weights for the mesh.
+   */
+  setWeights(weights: Vector4[] | null): void {
+    if (!this._accessible) {
+      throw "Not allowed to access data while accessible is false.";
+    }
+
+    if (weights.length !== this._vertexCount) {
+      throw "The array provided needs to be the same size as vertex count.";
+    }
+
+    this._vertexSlotChanged = weights != null;
+    this._vertexChangeFlag |= ValueChanged.Weight;
+    this._weights = weights;
+  }
+
+  /**
+   * Get weights for the mesh.
+   * @remarks Please call the setWeights() method after modification to ensure that the modification takes effect.
+   */
+  getWeights(): Vector4[] | null {
+    if (!this._accessible) {
+      throw "Not allowed to access data while accessible is false.";
+    }
+    return this._weights;
+  }
+
+  /**
+   * Set per-vertex joints for the mesh.
+   * @param joints - The joints for the mesh.
+   */
+  setJoints(joints: Vector4[] | null): void {
+    if (!this._accessible) {
+      throw "Not allowed to access data while accessible is false.";
+    }
+
+    if (joints.length !== this._vertexCount) {
+      throw "The array provided needs to be the same size as vertex count.";
+    }
+
+    this._vertexSlotChanged = !!this._joints !== !!joints;
+    this._vertexChangeFlag |= ValueChanged.Joint;
+    this._joints = joints;
+  }
+
+  /**
+   * Get joints for the mesh.
+   * @remarks Please call the setJoints() method after modification to ensure that the modification takes effect.
+   */
+  getJoints(): Vector4[] | null {
+    if (!this._accessible) {
+      throw "Not allowed to access data while accessible is false.";
+    }
+    return this._joints;
+  }
+
+  /**
+   * Set per-vertex tangents for the mesh.
+   * @param tangents - The tangents for the mesh.
+   */
+  setTangents(tangents: Vector4[] | null): void {
+    if (!this._accessible) {
+      throw "Not allowed to access data while accessible is false.";
+    }
+
+    if (tangents.length !== this._vertexCount) {
+      throw "The array provided needs to be the same size as vertex count.";
+    }
+
+    this._vertexSlotChanged = !!this._tangents !== !!tangents;
+    this._vertexChangeFlag |= ValueChanged.Tangent;
+    this._tangents = tangents;
+  }
+
+  /**
+   * Get tangents for the mesh.
+   * @remarks Please call the setTangents() method after modification to ensure that the modification takes effect.
+   */
+  getTangents(): Vector4[] | null {
+    if (!this._accessible) {
+      throw "Not allowed to access data while accessible is false.";
+    }
+    return this._tangents;
+  }
+
+  /**
+   * Set per-vertex uv for the mesh.
+   * @param uv - The uv for the mesh.
+   */
+  setUVs(uv: Vector2[] | null): void;
+  /**
+   * Set per-vertex uv for the mesh by channelIndex.
+   * @param uv - The uv for the mesh.
+   * @param channelIndex - The index of uv channels, in [0 ~ 7] range.
+   */
+  setUVs(uv: Vector2[] | null, channelIndex: number = 0): void {
```

Comment:
Function overloading will auto hide the original function, So you must write like this
```typescript
/**
   * Set per-vertex uv for the mesh.
   * @param uv - The uv for the mesh.
   */
  setUVs(uv: Vector2[] | null): void;
  /**
   * Set per-vertex uv for the mesh by channelIndex.
   * @param uv - The uv for the mesh.
   * @param channelIndex - The index of uv channels, in [0 ~ 7] range.
   */
 setUVs(uv: Vector2[] | null, channelIndex: number): void;

 setUVs(uv: Vector2[] | null, channelIndex: number = 0): void {
  //......
  }
```

Reply:
+1